src/miroil/gesture_ender.h

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright © Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MIROIL_GESTURE_ENDER_H
+#define MIROIL_GESTURE_ENDER_H
+
+#include <memory>
+
+namespace mir::frontend { class PointerInputDispatcher; }
+
+namespace miroil
+{
+void end_gesture();
+
+auto the_pointer_input_dispatcher() -> std::shared_ptr<mir::frontend::PointerInputDispatcher>;
```

Comment:
Should we rename this to avoid confusion with `mir::DefaultServerConfiguration`?

I mean, "gesture_ender_input_dispatcher()" or such?

Reply:
I think that would cause different confusion